process/peer/process.go

```diff
@@ -657,7 +706,7 @@ func (vs *validatorStatistics) updateValidatorInfo(validatorList []sharding.Vali
 			newRating = vs.rater.ComputeIncreaseValidator(shardId, peerAcc.GetTempRating())
 		case validatorFail:
 			peerAcc.DecreaseValidatorSuccessRate(1)
-			newRating = vs.rater.ComputeDecreaseValidator(shardId, peerAcc.GetTempRating())
+			newRating = vs.rater.ComputeIncreaseValidator(shardId, peerAcc.GetTempRating())
```

Comment:
case validatorSuccess:
case validatorFail:
 - the code.

Reply:
the successRates are different

Reply:
In both cases you call ComputeIncreateValidator with the same parameters.

Reply:
IncreaseValidatorSuccessRate on the first one and DecreaseValidatorSuccessRate on the second